src/composer.json

```diff
@@ -33,7 +33,7 @@
         "symfony/twig-bridge": "^6.0",
         "dompdf/dompdf": "^2.0.1",
         "symfony/polyfill-intl-idn" : "*",
-        "nelexa/zip": "^4.0.2"
```

Comment:
Why removing nelexa/zip isn't it been used any more?

Reply:
It was removed by an accident. Already reverted in https://github.com/FOSSBilling/FOSSBilling/pull/427/commits/658358e48140eb9bc8de40a57b0f2edc3700b485.